sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_utils.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from typing import cast
+from xml.etree.ElementTree import ElementTree
+
+
+import urllib.parse as urlparse
+
+from azure.servicebus.management import _constants as constants
+from ...management._handle_response_error import _handle_response_error
+
+
+async def extract_data_template(feed_class, convert, feed_element):
```

Comment:
Nit: Can we leave some comments for future devs as to the purpose of these functions?  It may be no fault of your own, the code isn't bad, it's just not immediately obvious from reading what the broader purpose is.  (You gave us a rundown on the call which was useful but rereading this with new eyes I wouldn't have that context)

Reply:
Good suggestion. Will add some comments here.

Reply:
Added detailed explanation